### PR TITLE
Allow PHP 8.1

### PR DESCRIPTION
**Pull request by snapshotpl**
|    Q          |   A
|-------------- | ------
| Documentation | no
| Bugfix        | no
| BC Break      | no
| New Feature   | no
| RFC           | no
| QA            | yes

### Description

Allow install on PHP 8.1

### CODE REVIEW

File: config/modules.config.php [hunk 1]
```diff
@@ -7,6 +7,8 @@
 declare(strict_types=1);
 
 return [
+    'Laminas\Mvc\I18n',
+    'Laminas\I18n',
```
Comment (Ocramius) [inline config/modules.config.php:11]: What's going on here? :thinking:

Comment (snapshotpl) [reply config/modules.config.php:11]: When run `composer install` there are missing modules (via composer plugin).
```
  Please select which config file you wish to inject 'Laminas\I18n' into:
  [0] Do not inject
  [1] config/modules.config.php
  [2] config/development.config.php.dist
  Make your selection (default is 1):

  Remember this option for other packages of the same type? (Y/n)
    Installing Laminas\I18n from package laminas/laminas-i18n
    Installing Laminas\Mvc\I18n from package laminas/laminas-mvc-i18n
```

**Comment by calvarezm70** [comment]
You need to update these other packages to make it work on php 8.1

         "laminas-api-tools/api-tools-admin": "^2.0.1",
         "laminas/laminas-developer-tools": "^2.3.0",
         "laminas/laminas-test": "^4.0",

**Comment by Ocramius** [comment]
@snapshotpl I can haz squash?

**Comment by snapshotpl** [comment]
@Ocramius done

**Review by Ocramius — APPROVED** [review]
Thanks @snapshotpl!

**Pull request merged**